src/site/content/en/blog/compat2021/index.md

```diff
@@ -0,0 +1,324 @@
+
+## Choosing what to focus on
+
+While there are browser compatibility issues in basically all of the web
+platform, the focus of this project is on a small number of the most problematic
+areas which can be made significantly better, thus removing them as top issues
+for developers.
+
+The compatibility project uses multiple criteria influencing which areas to
+prioritize, and some are:
+
++   Feature usage. For example, Flexbox is used in
+    [75%](https://www.chromestatus.com/metrics/feature/timeline/popularity/1692)
+    of all page views, and adoption is growing strongly in [HTTP
+    Archive](https://almanac.httparchive.org/en/2020/css#layout).
++   Number of bugs
+    (in [Chromium](https://bugs.chromium.org/p/chromium/issues/list),
+    [Gecko](https://bugzilla.mozilla.org/describecomponents.cgi),
+    [WebKit](https://bugs.webkit.org/)), and for Chromium, how many stars those
+    bugs have.
++   Survey results: 
+
+    +   [MDN DNA surveys](https://insights.developer.mozilla.org/)
+    +   [MDN Browser Compatibility Report](https://insights.developer.mozilla.org/reports/mdn-browser-compatibility-report-2020.html)
+    +   [State of CSS](https://2020.stateofcss.com/en-US/features/)
+        most known and used features
+
++   Test results from [web-platform-tests](https://github.com/web-platform-tests/wpt#the-web-platform-tests-project). For example, [Flexbox on
+    wpt.fyi](https://wpt.fyi/results/css/css-flexbox).
++   [Can I use](https://caniuse.com/)'s most-searched-for features.
+
+## The five top focus areas in 2021
+
+In 2020, Chromium started work addressing the top areas outlined in
+[Improving Chromium's browser compatibility in 2020](https://blog.chromium.org/2020/06/improving-chromiums-browser.html).
+In 2021, we are beginning a dedicated effort to go even further. Google and
+[Microsoft are working together on addressing top issues in Chromium](https://blogs.windows.com/msedgedev/2021/03/22/better-compatibility-compat2021/), along with [Igalia](https://www.igalia.com/). Igalia, who are regular contributors 
+to Chromium and WebKit, and maintainers of the official WebKit port for embedded devices, 
+have been very supportive and engaged in these compatibility efforts, and will be
+helping tackle and track the identified issues.
+
+Here are the areas which are committed to being fixed in 2021.
+
+### CSS Flexbox
+
+[CSS Flexbox](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout)
+is
+[widely used](https://www.chromestatus.com/metrics/feature/timeline/popularity/1692)
+on the web and there are still some major challenges for developers. For
+example, images as flex items have often been stretched incorrectly and
+differently between browsers. The issues with `auto-height` flex container in 
+[Chromium](https://bugs.chromium.org/p/chromium/issues/detail?id=721123) and [WebKit](https://bugs.webkit.org/show_bug.cgi?id=209983)
+used to have bugs, leading to incorrectly sized images.
```

Comment:
@foolip @robnyman This sentence reads funny ("the issues […] used to have bugs"). I'd suggest a patch but am not 100% sure what what was meant here. Was it: "Chromium and WebKit bugs with `auto-height` flex containers resulted in incorrectly-sized images"?

Reply:
Ah, I think this was a result of combining some language. I've sent https://github.com/GoogleChrome/web.dev/pull/4950 to simplify. Thanks!